src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -2068,6 +2068,39 @@ void ApplicationManagerImpl::ProcessServiceStatusUpdate(
       service_type, service_event, reason, app_id);
 
   rpc_service_->ManageHMICommand(notification);
+
+  if (hmi_apis::Common_ServiceEvent::REQUEST_REJECTED == service_event) {
+    StopServicesForApp(app, service_type);
+    state_ctrl_.SetRegularState(app,
```

Comment:
@IGapchuk better to use `StateControllerImpl::ExitDefaultWindow`

Reply:
@AKalinich-Luxoft I believe we can't use it because after setting regular state SDL should send to HMI CloseApplication Request for closing the specified application.